Explorer/Assets/Scripts/ProtobufPartialClasses/IDirtyMarker.cs

```diff
@@ -112,6 +112,21 @@ public partial class PBVideoPlayer : IDirtyMarker
         public bool IsDirty { get; set; }
     }
 
+    public partial class PBTween : IDirtyMarker
+    {
+        public bool IsDirty { get; set; }
+    }
+
+    public partial class PBTweenState : IDirtyMarker
+    {
+        public bool IsDirty { get; set; }
+    }
+
+    public partial class PBTweenSequence : IDirtyMarker
+    {
+        public bool IsDirty { get; set; }
+    }
```

Comment:
Let's remove this one since we are not going to use it for now

Reply:
done